package sample5;

public class ExceptionTestServiceOutput {

    //掛け算の結果＝積(product)
    private int product;

    public int getProduct() {
        return product;
    }

    public void setProduct(int product) {
        this.product = product;
    }
    
}
